src/const.ts

```diff
@@ -11,6 +11,8 @@ export const MOUSE_LEFT = "left";
 export const MOUSE_RIGHT = "right";
 export const MOUSE_MIDDLE = "middle";
 
+export const VELOCITY_INTERVAL = 20;
```

Comment:
Why did you set this value `20`?
Isn't the frame delta usually 16ms(1/60) when 60fps?

Reply:
In the past version of Axes, it was working based on 25ms, but this part felt unnatural and was reduced.
16ms(1/60) equivalent to 60fps seems to be a more suitable constant.